core/vnl/CMakeLists.txt

```diff
@@ -16,18 +16,6 @@ option(VNL_CONFIG_THREAD_SAFE
   "Whether thread-safe vnl implementations are used." ON)
 
 
-#if( VXL_HAS_EMMINTRIN_H AND VXL_HAS_SSE2_HARDWARE_SUPPORT )
-#  option(VNL_CONFIG_ENABLE_SSE2
-#    "Enable Streaming SIMD Extensions 2 optimisations (hardware dependant)." ON)
-#else()
-  option(VNL_CONFIG_ENABLE_SSE2
-    "Enable Streaming SIMD Extensions 2 optimisations (hardware dependant). Currently broken. For use by VNL developers only." OFF)
-#endif()
-if(VNL_CONFIG_ENABLE_SSE2)
-  # Tested on ubuntu and Mac.  ctest becomes unstable and failures change between runs.
-  message(FATAL_ERROR  "VNL_CONFIG_ENABLE_SSE2 option currently fails testing on all platforms,  this is not suitable for use at the momemnt.")
```

Comment:
Feature has been forced unusable since 2018, and was functionally unusable for many years prior to that.